### ISSUE TITLE
no-qualifying-elements - allow-element-with-class: true bug?

### ISSUE THREAD

**Issue by philwareham** [sasstools/sass-lint#443]
I have this in my sass-lint.yml file:

    no-qualifying-elements:
      - 1
      - allow-element-with-class: true

Yet am still getting the warning message 'Qualifying elements are not allowed for class selectors'.

`allow-element-with-attribute` and `allow-element-with-id` work as expected.

**Comment by DanPurdy** [comment]
HI @philwareham, thanks for the report.

Could you provide your version of sass-lint and possibly a sample of the code that is failing so we can investigate a little further. 

Thanks

**Comment by philwareham** [comment]
Sure, it's sass-lint version 1.3.3 and a sample of Sass throwing the warning is as follows:

    span.disabled { ... content ... }

**Comment by DanPurdy** [comment]
Hmm, I can't seem to replicate this on 1.3.3 I'm afraid, is there something else around it that changes its context?
<img width="398" alt="screen shot 2015-12-08 at 09 23 59" src="https://cloud.githubusercontent.com/assets/1386054/11652098/96b721ce-9d8d-11e5-9437-84621ec52bd8.png">

<img width="675" alt="screen shot 2015-12-08 at 09 23 47" src="https://cloud.githubusercontent.com/assets/1386054/11652097/93101d6e-9d8d-11e5-9be1-0b0252120492.png">

<img width="393" alt="screen shot 2015-12-08 at 09 25 37" src="https://cloud.githubusercontent.com/assets/1386054/11652103/ab5cffcc-9d8d-11e5-811b-3747b630eff0.png">
<img width="658" alt="screen shot 2015-12-08 at 09 24 14" src="https://cloud.githubusercontent.com/assets/1386054/11652105/afa30054-9d8d-11e5-842b-54864b8444ba.png">

**Comment by philwareham** [comment]
OK, I've tested further, and when like this it works:

    no-qualifying-elements:
      - 1
      - allow-element-with-class: true

When like this (which is actually what is in my lint file), it doesn't work:

    no-qualifying-elements:
      - 1
      - allow-element-with-attribute: true
      - allow-element-with-class: true
      - allow-element-with-id: false

**Comment by philwareham** [comment]
Ah, I've just realised I have the syntax wrong (doesn't need the dashes). Really sorry to waste your time!

**Comment by DanPurdy** [comment]
No problem. Glad you sorted it.

For anyone else coming across this issue you need to pass the multiple options through as an object like so.

```yml
  no-qualifying-elements:
    - 1
    -
      allow-element-with-attribute: true
      allow-element-with-class: false
      allow-element-with-id: true
```
